### ISSUE TITLE
Encounter error with example

### ISSUE THREAD

**Issue by beiller** [DevPoodle/compute-shader-plus#7]
I had to add a flag to `create_rd_texture_format`

```	
texture_format.usage_bits = (
		RenderingDevice.TEXTURE_USAGE_SAMPLING_BIT +
		RenderingDevice.TEXTURE_USAGE_STORAGE_BIT +
		RenderingDevice.TEXTURE_USAGE_CAN_UPDATE_BIT +
		RenderingDevice.TEXTURE_USAGE_CAN_COPY_FROM_BIT
	)
```

Maybe is it because of my setup?

**Comment by DevPoodle** [comment]
Thanks for opening an issue! I believe this was fixed in a previous pr (#5), but the version in the asset library doesn't have it yet. I'll try to get it updated soon.

**Comment by DevPoodle** [comment]
Alright, this fix should be in the asset library version now! ❤️
